src/utils/translate/en.json

```diff
@@ -448,8 +449,17 @@
   },
   "StudyInformation": "Study Information",
   "StudyInformationTooltip": {
-    "FirstParagraph": "Filter on study details, including source type (publication, preprint, news, or report) and study status (ongoing, completed).",
-    "SecondParagraph": "Risk of bias: Reflects the extent to which the true prevalence may be different from the estimated prevalence. Estimated by SeroTracker reviewers based on the Joanna Briggs Institute critical appraisal tool for prevalence estimates."
+    "FirstParagraph": "Filter on study details, including source type (publication, preprint, news, or report), study status (ongoing, completed), and alignment with the WHO Unity protocol.",
+    "SecondParagraph": {
+      "PartOne": "Risk of bias: ",
+      "PartTwo": "Reflects the extent to which the true prevalence may be different from the estimated prevalence. Estimated by SeroTracker reviewers based on the Joanna Briggs Institute critical appraisal tool for prevalence estimates."
+    },
+    "ThirdParagraph": {

```

Comment:
I know we're overhauling translation soon, but think we should not include extra spaces at the end of text in this JSON, that should be handled elsewhere.